src/routes/docs/products/auth/+layout.svelte

```diff
@@ -36,6 +36,10 @@
                     label: 'Teams',
                     href: '/docs/products/auth/teams'
                 },
+                {
+                    label: 'Preferences',
+                    href: '/docs/products/auth/preferences'
+                },
```

Comment:
When the docs were first designed/structured, Concepts were used to explain aspects in Appwrite while Journeys talked more about use cases. See our [style](https://github.com/appwrite/website/blob/main/STYLE.md#products) guide: 

<img width="997" alt="image" src="https://github.com/user-attachments/assets/c1416240-9239-40db-b9cd-884ad2f09bec" />


Given that, I would recommend splitting up the content so that this new Preferences content goes under Journeys.

If we want Preferences more discoverable, we should move the Preferences in Accounts into its own page and add a section on Team Preferences.